### PR TITLE
Feature/sdka 7

### PR DESCRIPTION
**Pull request by TrayEdge**
## We tested it with test exceptions throws and we was able to observe them inside Sentry Dashboard.

### CODE REVIEW

**Review by dim-ser — APPROVED** [review]
Maybe we can move "https://f3ca34981162465cabb2783483179ae9@o4504125304209408.ingest.sentry.io/4504140012584960" to the config class as a constant and reuse it in different places ?
But it's just an advice.

**Review by IvanK88 — CHANGES_REQUESTED** [review]

File: Client/src/main/java/com/seamlesspay/api/SeamlesspayFragment.java [hunk 1]
```diff
@@ -227,6 +230,12 @@ public void onCreate(Bundle savedInstanceState) {
     }
   }
 
+  private void setUpSentry() {
```
[the remark this answers is not in the document]
Comment (TrayEdge) [reply Client/src/main/java/com/seamlesspay/api/SeamlesspayFragment.java:233]: The problem is that this code is used three times in different modules which are independent and can be used separately. This is why we can not share it. The only solution for now would be to create a separate shared module and put this method there. But, we will need to add Sentry dependency into this module. Maybe, in a future when we will have more additional configs we can create a shared module to put everything there.

File: CardForm/src/main/java/com/seamlesspay/cardform/view/CardForm.java [hunk 2]
```diff
@@ -170,6 +174,18 @@ private void init() {
     mCardNumber.setOnCardTypeChangedListener(this);
   }
 
+  private void setUpSentry() {
+    SentryOptions options = new SentryOptions();
+    options.setDsn("https://f3ca34981162465cabb2783483179ae9@o4504125304209408.ingest.sentry.io/4504140012584960");
```
[the remark this answers is not in the document]
Comment (TrayEdge) [reply CardForm/src/main/java/com/seamlesspay/cardform/view/CardForm.java:179]: Not at this point, the situation is the same as with comment below - this part of code is used in separate modules. We can move this string as a constant to the separate file, but in this case we will need to create three files for each module or create a shared module. This is something that we can discuss later in terms of our future plans for the library itself.

**Comment by TrayEdge** [comment]
> Maybe we can move "https://f3ca34981162465cabb2783483179ae9@o4504125304209408.ingest.sentry.io/4504140012584960" to the config class as a constant and reuse it in different places ?
> But it's just an advice.

This is a good point, we will definitely do it in the future, for now all the different places where this string is used are in a separate modules and can not be accessed from each other. Please, check my other replays.

**Review by IvanK88 — APPROVED** [review]

**Pull request merged**